packages/config/config/devices/0x019b/4512688.json

```diff
@@ -18,6 +18,18 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Smoke Sensor (Basic Set)",
+			"description": "When the smoke detector is active, it will send Basic Set (0xFF) to this group. When the smoke detector is deactived, it will send Basic Set (0x00).",
```

Comment:
Remove the description from the groups. None of the UIs display it anyway. Defining the groups isn't actually required for most devices but doing so is fine.

Reply:
removed description  from group 2